*** main.R	2021-07-20 11:25:38.259746017 -0400
--- main_win.R	2021-07-20 11:31:07.257197194 -0400
***************
*** 4,9 ****
--- 4,11 ----
  stopifnot(packageVersion("pomp")>="3.0")
  set.seed(1350254336)
  
+ options(pomp_cdir="./tmp")
+ 
  source("https://kingaa.github.io/sbied/pfilter/model.R")
  
  
***************
*** 44,49 ****
--- 47,57 ----
  ## for an explanation.
  bake(file="local_search.rds",{
    registerDoRNG(482947940)
+   measSIR %>%
+     pomp(
+       partrans=parameter_trans(log="Beta",logit=c("rho","eta")),
+       paramnames=c("Beta","rho","eta")
+     ) -> measSIR
    foreach(i=1:20,.combine=c) %dopar% {
      library(pomp)
      library(tidyverse)
***************
*** 51,59 ****
        mif2(
          Np=2000, Nmif=50,
          cooling.fraction.50=0.5,
!         rw.sd=rw.sd(Beta=0.02, rho=0.02, eta=ivp(0.02)),
!         partrans=parameter_trans(log="Beta",logit=c("rho","eta")),
!         paramnames=c("Beta","rho","eta")
        )
    } -> mifs_local
    attr(mifs_local,"ncpu") <- getDoParWorkers()
--- 59,65 ----
        mif2(
          Np=2000, Nmif=50,
          cooling.fraction.50=0.5,
!         rw.sd=rw.sd(Beta=0.02, rho=0.02, eta=ivp(0.02))
        )
    } -> mifs_local
    attr(mifs_local,"ncpu") <- getDoParWorkers()
***************
*** 73,90 ****
  
  
  bake(file="lik_local.rds",{
!   registerDoRNG(900242057)
!   foreach(mf=mifs_local,.combine=rbind) %dopar% {
!     library(pomp)
!     library(tidyverse)
!     evals <- replicate(10, logLik(pfilter(mf,Np=5000)))
!     ll <- logmeanexp(evals,se=TRUE)
!     mf %>% coef() %>% bind_rows() %>%
!       bind_cols(loglik=ll[1],loglik.se=ll[2])
!   } -> results
!   attr(results,"ncpu") <- getDoParWorkers()
!   results
! }) -> results
  t_local <- attr(results,"system.time")
  ncpu_local <- attr(results,"ncpu")
  
--- 79,96 ----
  
  
  bake(file="lik_local.rds",{
!     registerDoRNG(900242057)
!     foreach(mf=mifs_local,.combine=rbind) %dopar% {
!       library(pomp)
!       library(tidyverse)
!       evals <- replicate(10, logLik(pfilter(mf,Np=5000)))
!       ll <- logmeanexp(evals,se=TRUE)
!       mf %>% coef() %>% bind_rows() %>%
!         bind_cols(loglik=ll[1],loglik.se=ll[2])
!     } -> results
!     attr(results,"ncpu") <- getDoParWorkers()
!     results
!   }) -> results
  t_local <- attr(results,"system.time")
  ncpu_local <- attr(results,"ncpu")
  
***************
*** 267,291 ****
  mf1 <- mifs_local[[1]]
  bake(file="rho_profile.rds",
    dependson=guesses,{
!     registerDoRNG(2105684752)
!     foreach(guess=iter(guesses,"row"), .combine=rbind) %dopar% {
!       library(pomp)
!       library(tidyverse)
!       mf1 %>%
!         mif2(params=guess,
!           rw.sd=rw.sd(Beta=0.02,eta=ivp(0.02))) %>%
!         mif2(Nmif=100,cooling.fraction.50=0.3) %>%
!         mif2() -> mf
!       replicate(
!         10,
!         mf %>% pfilter(Np=5000) %>% logLik()) %>%
!         logmeanexp(se=TRUE) -> ll
!       mf %>% coef() %>% bind_rows() %>%
!         bind_cols(loglik=ll[1],loglik.se=ll[2])
!     } -> results
!     attr(results,"ncpu") <- getDoParWorkers()
!     results
!   }) -> results
  t_rho <- attr(results,"system.time")
  ncpu_rho <- attr(results,"ncpu")
  read_csv("measles_params.csv") %>%
--- 273,297 ----
  mf1 <- mifs_local[[1]]
  bake(file="rho_profile.rds",
    dependson=guesses,{
!   registerDoRNG(2105684752)
!   foreach(guess=iter(guesses,"row"), .combine=rbind) %dopar% {
!     library(pomp)
!     library(tidyverse)
!     mf1 %>%
!       mif2(params=guess,
!         rw.sd=rw.sd(Beta=0.02,eta=ivp(0.02))) %>%
!       mif2(Nmif=100,cooling.fraction.50=0.3) %>%
!       mif2() -> mf
!     replicate(
!       10,
!       mf %>% pfilter(Np=5000) %>% logLik()) %>%
!       logmeanexp(se=TRUE) -> ll
!     mf %>% coef() %>% bind_rows() %>%
!       bind_cols(loglik=ll[1],loglik.se=ll[2])
!   } -> results
!   attr(results,"ncpu") <- getDoParWorkers()
!   results
! }) -> results
  t_rho <- attr(results,"system.time")
  ncpu_rho <- attr(results,"ncpu")
  read_csv("measles_params.csv") %>%
***************
*** 333,367 ****
  
  bake(file="global_search2.rds",
    dependson=guesses,{
!     registerDoRNG(610408798)
!     foreach(guess=iter(guesses,"row"), .combine=rbind) %dopar% {
!       library(pomp)
!       library(tidyverse)
!       measSIR %>%
!         mif2(params=guess, Np=2000, Nmif=100,
!           cooling.fraction.50=0.5,
!           partrans=parameter_trans(
!             log=c("Beta","mu_IR"),
!             logit="eta"), paramnames=c("Beta","mu_IR","eta"),
!           rw.sd=rw.sd(Beta=0.02,mu_IR=0.02,eta=ivp(0.02))) -> mf
!       mf %>%
!         mif2(
!           Nmif=100,rw.sd=rw.sd(Beta=0.01,mu_IR=0.01,eta=ivp(0.01))
!         ) %>%
!         mif2(
!           Nmif=100,
!           rw.sd=rw.sd(Beta=0.005,mu_IR=0.005,eta=ivp(0.005))
!         ) -> mf
!       replicate(
!         10,
!         mf %>% pfilter(Np=5000) %>% logLik()
!       ) %>% logmeanexp(se=TRUE) -> ll
!       mf %>% coef() %>% bind_rows() %>%
!         bind_cols(loglik=ll[1],loglik.se=ll[2])
!     } -> results
!     attr(results,"ncpu") <- getDoParWorkers()
!     results
!   }) %>%
    filter(is.finite(loglik)) -> results
  t_expglob <- attr(results,"system.time")
  ncpu_expglob <- attr(results,"ncpu")
--- 339,378 ----
  
  bake(file="global_search2.rds",
    dependson=guesses,{
!   registerDoRNG(610408798)
!   measSIR %>%
!     pomp(
!       partrans=parameter_trans(
!         log=c("Beta","mu_IR"),
!         logit="eta"
!       ),
!       paramnames=c("Beta","mu_IR","eta")
!     ) -> measSIR
!   foreach(guess=iter(guesses,"row"), .combine=rbind) %dopar% {
!     library(pomp)
!     library(tidyverse)
!     measSIR %>%
!       mif2(params=guess, Np=2000, Nmif=100,
!         cooling.fraction.50=0.5,
!         rw.sd=rw.sd(Beta=0.02,mu_IR=0.02,eta=ivp(0.02))) -> mf
!     mf %>%
!       mif2(
!         Nmif=100,rw.sd=rw.sd(Beta=0.01,mu_IR=0.01,eta=ivp(0.01))
!       ) %>%
!       mif2(
!         Nmif=100,
!         rw.sd=rw.sd(Beta=0.005,mu_IR=0.005,eta=ivp(0.005))
!       ) -> mf
!     replicate(
!       10,
!       mf %>% pfilter(Np=5000) %>% logLik()
!     ) %>% logmeanexp(se=TRUE) -> ll
!     mf %>% coef() %>% bind_rows() %>%
!       bind_cols(loglik=ll[1],loglik.se=ll[2])
!   } -> results
!   attr(results,"ncpu") <- getDoParWorkers()
!   results
! }) %>%
    filter(is.finite(loglik)) -> results
  t_expglob <- attr(results,"system.time")
  ncpu_expglob <- attr(results,"ncpu")
***************
*** 412,437 ****
  
  bake(file="mu_IR_profile1.rds",
    dependson=guesses,{
!     registerDoRNG(610408798)
!     foreach(guess=iter(guesses,"row"), .combine=rbind) %dopar% {
!       library(pomp)
!       library(tidyverse)
!       measSIR %>%
!         mif2(params=guess, Np=2000, Nmif=100,
!           partrans=parameter_trans(log="Beta",logit="eta"),
!           paramnames=c("Beta","eta"), cooling.fraction.50=0.5,
!           rw.sd=rw.sd(Beta=0.02,eta=ivp(0.02))
!         ) %>% mif2(Nmif=100) %>%
!         mif2(Nmif=100,rw.sd=rw.sd(Beta=0.01,eta=ivp(0.01))) %>%
!         mif2(Nmif=100,rw.sd=rw.sd(Beta=0.005,eta=ivp(0.005))) -> mf
!       replicate(10,mf %>% pfilter(Np=5000) %>% logLik()) %>%
!         logmeanexp(se=TRUE) -> ll
!       mf %>% coef() %>% bind_rows() %>%
!         bind_cols(loglik=ll[1],loglik.se=ll[2])
!     } -> results
!     attr(results,"ncpu") <- getDoParWorkers()
!     results
!   }) %>%
    filter(is.finite(loglik)) -> results
  t_muIR <- attr(results,"system.time")
  ncpu_muIR <- attr(results,"ncpu")
--- 423,452 ----
  
  bake(file="mu_IR_profile1.rds",
    dependson=guesses,{
!   registerDoRNG(610408798)
!   measSIR %>%
!     pomp(
!       partrans=parameter_trans(log="Beta",logit="eta"),
!       paramnames=c("Beta","eta")
!     ) -> measSIR
!   foreach(guess=iter(guesses,"row"), .combine=rbind) %dopar% {
!     library(pomp)
!     library(tidyverse)
!     measSIR %>%
!       mif2(params=guess, Np=2000, Nmif=100,
!         cooling.fraction.50=0.5,
!         rw.sd=rw.sd(Beta=0.02,eta=ivp(0.02))
!       ) %>% mif2(Nmif=100) %>%
!       mif2(Nmif=100,rw.sd=rw.sd(Beta=0.01,eta=ivp(0.01))) %>%
!       mif2(Nmif=100,rw.sd=rw.sd(Beta=0.005,eta=ivp(0.005))) -> mf
!     replicate(10,mf %>% pfilter(Np=5000) %>% logLik()) %>%
!       logmeanexp(se=TRUE) -> ll
!     mf %>% coef() %>% bind_rows() %>%
!       bind_cols(loglik=ll[1],loglik.se=ll[2])
!   } -> results
!   attr(results,"ncpu") <- getDoParWorkers()
!   results
! }) %>%
    filter(is.finite(loglik)) -> results
  t_muIR <- attr(results,"system.time")
  ncpu_muIR <- attr(results,"ncpu")
